### ISSUE TITLE
Use pull request templating to ensure that PR only edits the Rmd files

### ISSUE THREAD

**Issue by fmichonneau** [datacarpentry/R-ecology-lesson#61]
Use GitHub's new functionality to ensure, when someone opens a PR, that only the Rmd file gets submitted. See https://github.com/blog/2111-issue-and-pull-request-templates

**Comment by benmarwick** [comment]
I had a quick look into this, and it seems that all we can do is provide a template for the issue that is posted with the PR. I couldn't see any way to enforce rules about PRs, except to get confirmation from the request, for example, something like this:

```
Refers to #[insert issue number].

If this is a PR for a lesson, please confirm:
[ ] I have edited the Rmd file and not the html file

Changes proposed in this pull request:

-    
-   
-   

@fmichonneau @aurielfournier
```

Would that be useful?

**Comment by fmichonneau** [comment]
Yes, it's what I had in mind. It was tried briefly with SWC R inflammation lesson, but it was just at the time when we transitioned to the new template and this got lost in the process. I think it'd be worth a try to see if it is useful or if it makes things more confusing. Lately, I noticed that we received few PRs that included the html file...

For reference, here is the text I had submitted at the time for the SWC repo:

```
Thank you for submitting a pull request! If you are proposing a change to one of the lessons,
please make sure that:

[ ] You are only committing the `.Rmd` file (the `md` and `html` files will be
generated by the lesson maintainer)
[ ] If you are providing a new challenge, you are also providing the solution
[ ] You are following the correct formatting
```

It seems that adding a mention to review `CONTRIBUTING.md` that should be linked at the top of the page automatically by GitHub might also be useful.
